django/db/models/query.py

```diff
@@ -1073,19 +1073,28 @@ def _batched_insert(self, objs, fields, batch_size):
                 self._insert(item, fields=fields, using=self.db)
         return inserted_ids
 
-    def _clone(self, **kwargs):
-        query = self.query.clone()
-        if self._sticky_filter:
-            query.filter_is_sticky = True
-        clone = self.__class__(model=self.model, query=query, using=self._db, hints=self._hints)
-        clone._for_write = self._for_write
-        clone._prefetch_related_lookups = self._prefetch_related_lookups
-        clone._known_related_objects = self._known_related_objects
-        clone._iterable_class = self._iterable_class
-        clone._fields = self._fields
-
-        clone.__dict__.update(kwargs)
-        return clone
+    def _chain(self, **kwargs):
+        """"Prepare for another QuerySet operation."""
```

Comment:
This is four " instead of 3. Docstring would probably also be better explaining it returns a clone, e.g. "Create a clone of the current QuerySet ready for another operation" ?